### PR TITLE
Samsung.SmartView version 1.0.0.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98678)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

This software does not appear to be provided by the publisher anymore. Packages must come from the publisher's website. Based on this, the package might not be a good fit for installation through the WinGet package manager.

**Comment by upintheairsheep** [comment]
> Hi @upintheairsheep,
> 
> This software does not appear to be provided by the publisher anymore. Packages must come from the publisher's website. Based on this, the package might not be a good fit for installation through the WinGet package manager.

https://www.samsung.com/levant/apps/tv/smart-view/ Still avalible.

File: manifests/s/Samsung/SmartView/1.0.0.0/Samsung.SmartView.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SmartView
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Smart View
```
Comment (stephengillie) [inline manifests/s/Samsung/SmartView/1.0.0.0/Samsung.SmartView.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.samsung.com/
PackageName: Smart View
PackageUrl: https://www.samsung.com/levant/apps/tv/smart-view/
```

**Comment by stephengillie** [comment]
> https://www.samsung.com/levant/apps/tv/smart-view/ Still avalible.

Ah, thanks.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**